include/libdnf/rpm/package.hpp

```diff
@@ -38,280 +38,435 @@ class Goal;
+    // @replaces libdnf:libdnf/hy-package.h:function:dnf_package_get_hdr_end(DnfPackage * pkg)
+    unsigned long long get_hdr_end() const;
 
-    // SYSTEM
+    /// @return Number of media (usually a CD/DVD disc) on which the package is located
+    /// @since 5.0
+    //
+    // @replaces dnf:dnf/package.py:attribute:Package.medianr
+    // @replaces libdnf:libdnf/hy-package.h:function:dnf_package_get_medianr(DnfPackage * pkg)
+    unsigned long long get_media_number() const;
 
-    /// @replaces libdnf:libdnf/hy-package.h:function:dnf_package_installed(DnfPackage * pkg)
-    /// @replaces dnf:dnf/package.py:attribute:Package.installed
+    // ===== SYSTEM =====
+
+    /// @return RPM package location (file path) on the file system:
+    ///         * If the package is a local package, return its path
+    ///         * If the package is a remote package downloaded to cache, return path to the cache
+    ///         * If the package is a remote package that hasn't been downloaded yet, return path to the cache
+    /// @since 5.0
+    //
+    // @replaces dnf:dnf/package.py:attribute:Package.localPkg
+    // @replaces libdnf:libdnf/hy-package.h:function:dnf_package_get_local_baseurl(DnfPackage * pkg)
+    std::string get_local_path() const;
+
+    /// @return True if the package is installed on the system, false otherwise
+    /// @since 5.0
+    //
+    // @replaces libdnf:libdnf/hy-package.h:function:dnf_package_installed(DnfPackage * pkg)
+    // @replaces dnf:dnf/package.py:attribute:Package.installed
     bool is_installed() const;
 
     /// TODO is_local
-    /// @replaces dnf:dnf/package.py:method:Package.localPkg(self)
-    /// @replaces libdnf:libdnf/dnf-package.h:function:dnf_package_is_local(DnfPackage * pkg)
+    // @replaces dnf:dnf/package.py:method:Package.localPkg(self)
+    // @replaces libdnf:libdnf/dnf-package.h:function:dnf_package_is_local(DnfPackage * pkg)
     //bool is_local() const;
 
     /// TODO get_from_repo_id - requires swdb
     /// For an installed package, return repoid of repo from the package was installed.
     /// For an available package, return an empty string.
-    ///
-    /// @replaces dnf:dnf/package.py:attribute:Package.ui_from_repo
-    /// @replaces libdnf:libdnf/dnf-package.h:function:dnf_package_get_origin(DnfPackage * pkg)
+    //
+    // @replaces dnf:dnf/package.py:attribute:Package.ui_from_repo
+    // @replaces libdnf:libdnf/dnf-package.h:function:dnf_package_get_origin(DnfPackage * pkg)
     // void get_from_repo_id() const;
 
-    /// @replaces dnf:dnf/package.py:attribute:Package.hdr_end
-    /// @replaces libdnf:libdnf/hy-package.h:function:dnf_package_get_hdr_end(DnfPackage * pkg)
-    unsigned long long get_hdr_end() const;
-
-    /// @replaces dnf:dnf/package.py:attribute:Package.installtime
-    /// @replaces libdnf:libdnf/hy-package.h:function:dnf_package_get_installtime(DnfPackage * pkg)
+    /// @return The unix timestamp (seconds since epoch) when a package was installed.
```

Comment:
> The unix timestamp (seconds since epoch) **of** when **the** package was installed.

Not entirely sure of the addition of "of" :grin: but I think it's better.

Reply:
Fixed.